### PR TITLE
Fix doc warnings (and bugs)

### PR DESCRIPTION
**Pull request by sloriot**
Fix warnings and bugs discovered by doxygen improvements

### CODE REVIEW

**Comment by lrineau** [comment]
Bingo!! 
https://cgal.geometryfactory.com/CGAL/Manual_doxygen_test/

File: Jet_fitting_3/doc/Jet_fitting_3/Jet_fitting_3.txt [hunk 1]
```diff
@@ -224,7 +224,7 @@ In addition, the class `Monge_via_jet_fitting` stores
 This concept provides the types for the input sample points, together
 with \f$ 3d\f$ vectors and a number type. It is used as template for the
 class `Monge_via_jet_fitting<DataKernel, LocalKernel, SvdTraits>`. Typically, one can use
-`Cartesian<double>`.
+`Cartesian< double >`.
```
Comment (lrineau) [inline Jet_fitting_3/doc/Jet_fitting_3/Jet_fitting_3.txt]: Why does that matter? If it does, maybe we should explain why in the guidelines.

**Comment by lrineau** [comment]
Successfully tested in [CGAL-4.13-Ic-238][*], and then merged.

  [*]: https://cgal.geometryfactory.com/CGAL/testsuite/results-4.13-Ic-238.shtml

File: Jet_fitting_3/doc/Jet_fitting_3/Jet_fitting_3.txt (hunk 1 above)
Comment (sloriot) [reply Jet_fitting_3/doc/Jet_fitting_3/Jet_fitting_3.txt]: It was understood as a html tag by doxygen.

**Pull request merged**